test/collections/observable_list_test.dart

```diff
@@ -0,0 +1,444 @@
+import 'dart:async';
+
+import 'package:observable/observable.dart';
+import 'package:test/test.dart';
+
+main() {
+  group('$ObservableList', () {
+    group('list api', _runListTests);
+    _runObservableListTests();
+    _runDeprecatedTests();
+  });
+}
+
+bool _hasListChanges(List<ChangeRecord> c) {
+  return c.any((r) => r is ListChangeRecord);
```

Comment:
[nit] Why choose return here and => in _onlyListChanges? How about => for all of them?